data/en/plugins_list.json

```diff
@@ -850,12 +850,6 @@
     "Icon": "N/A",
     "Description": "Sink plugin to send the messages from the pipeline to an external http endpoint."
   },
-  "DynamicMultiADLS": {
```

Comment:
I think this is a different plugin, which is supported. It shouldn't be removed.

Reply:
done